gabrielaEnache/src/main/java/collections/store/Address.java

```diff
@@ -0,0 +1,46 @@
+package collections.store;
+
+public class Address {
+    public String country;
+    public String city;
+    public String street;
+
+    public Address(String country, String city, String street) {
+        this.country = country;
+        this.city = city;
+        this.street = street;
+    }
+
+    public String getCountry() {
```

Comment:
getters and setters are used in order to access the Object attributes, in our case they are public, therefore you can access them directly without using getters or setters. I would like to recommenced to make the properties(attributes) private. Which means that they will be accessible only through the getters and setters methods.

Reply:
Changed fields to private